Makefile

```diff
@@ -28,10 +31,17 @@ build: doc dg
 	docker build --tag=${IMAGE_REPOSITORY} -f $(DOCKERFILE) .
 
 run: build
-	docker run -e MYHOSTNAME=localhost $(IMAGE_REPOSITORY)
+	docker run -e MYHOSTNAME=localhost -e PLAIN_AUTH $(IMAGE_REPOSITORY)
 
 test: build
-	cd tests; MODULE=docker URL="docker=${IMAGE_REPOSITORY}" DOCKERFILE="../$(DOCKERFILE)" VERSION=${VERSION} DISTRO=${DISTRO} mtf -l *.py
+	cd tests; MODULE=docker URL="docker=${IMAGE_REPOSITORY}" DOCKERFILE="../$(DOCKERFILE)" VERSION=${VERSION} DISTRO=${DISTRO} mtf-env-set
+	cd tests; MODULE=docker URL="docker=${IMAGE_REPOSITORY}" DOCKERFILE="../$(DOCKERFILE)" VERSION=${VERSION} DISTRO=${DISTRO} mtf --show-job-log *.py
```

Comment:
please remove `--show-job-log` option -> it leads to very long output not easy, to parse it by human.
and you've removed `-l` option of mtf it is intetional or by mistake? or because this caused that log were too long?